.github/workflows/release.yml

```diff
@@ -52,3 +55,20 @@ jobs:
           refreshToken: ${{ secrets.OAUTH_REFRESH_TOKEN }}
           force: true # We have checked the `dart pub publish --dry-run` `in build.yaml`, it's ok to force publish here.
           skipTests: true
+      - uses: subosito/flutter-action@v2
```

Comment:
Same above. Can we use the `dart-lang/setup-dart` instead of the flutter one to reduce the setup time of the flutter?